libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/roadobject/reststop/RestStop.kt

```diff
@@ -11,6 +11,7 @@ import com.mapbox.navigation.base.trip.model.roadobject.location.RoadObjectLocat
  * @param restStopType information about a rest stop. See [RestStopType].
  * @param name name of the rest stop.
  * @param amenities facilities associated with the rest stop. See [AmenityType]
+ * @param guideMapUri URI to guide map image. To use it you must add an access token
```

Comment:
```suggestion
 * @param guideMapUri URI to guide map image. To be able to access the resource, you need to append an `access_token=` query parameter. Data availability may vary.
```

Reply:
`?access_token=`

Reply:
Depends on where in the query it lands :upside_down_face: